roles/ood_user_reg_cloud/tasks/main.yml

```diff
@@ -42,7 +42,7 @@
 - name: Create log directory
   file:
     path: "{{ user_register_app_path }}/log"
-    owner: "{{ RegUser_app_user }}"
+    owner: apache
     group: apache
```

Comment:
We should use a variable for the user and group owners since this may need to change.  The variable should be the same as the process owner in the .service configuration.